PennMobileShared/Dining/DiningAPI.swift

```diff
@@ -10,11 +10,16 @@ import SwiftyJSON
 import Foundation
 
 public class DiningAPI {
+    
+    public static let directory = "diningVenue-v2.json"
```

Comment:
Was there a particular reason this was removed from `DiningVenue`?

Reply:
lol no, moving back my bad